### PR TITLE
Allow to pass callables to EventHandler (not only Closure)

### PR DESCRIPTION
**Pull request by tomaszdurka**

### CODE REVIEW

**Comment by tomaszdurka** [comment]
@dlondero please review

**Comment by dlondero** [comment]
@tomaszdurka please apply CS

**Comment by tomaszdurka** [comment]
Hmm? Codestyles? I think they are fine.

File: library/CM/EventHandler/EventHandlerTrait.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+trait CM_EventHandler_EventHandlerTrait {
+
+    /**
+     * @var array[] $_callbacks
+     */
+    private $_callbacks = array();
+
+    /**
+     * @param string  $event
```
Comment (dlondero) [inline library/CM/EventHandler/EventHandlerTrait.php]: CS here :)

**Comment by tomaszdurka** [comment]
@dlondero please take a look again, thanks.

**Comment by dlondero** [comment]
@tomaszdurka lgtm

**Pull request merged**